react-components/src/components/BundleProductOptions/__test__/bundleProductOptions.test.js

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ *
+ *    Copyright 2020 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { MockedProvider } from '@apollo/react-testing';
+import { render } from '@testing-library/react';
+import * as ConfigContext from '../../../context/ConfigContext';
+
+import i18n from '../../../../__mocks__/i18nForTests';
+
+import BundleProductOptions from '../bundleProductOptions';
+
+describe('<BundleProductOptions>', () => {
+    beforeAll(() => {
+        // mock createPortal because we don't have the DOM element to render the BundleProductOptions
+        jest.spyOn(ConfigContext, 'useConfigContext').mockImplementation(() => {
+            return {
+                mountingPoints: { bundleProductOptionsContainer: '#bundle-product-options' }
+            };
+        });
+    });
+
+    it('renders the component with no sku', () => {
+        const { asFragment } = render(
+            <I18nextProvider i18n={i18n}>
+                <MockedProvider>
+                    <BundleProductOptions />
+                </MockedProvider>
+            </I18nextProvider>
+        );
+
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    it('renders the component with sku', () => {
+        jest.spyOn(document, 'querySelector').mockImplementation(selector => {
```

Comment:
You could just add an outer `<div id="bundle-product-options" data-sku="sku">` to the render method.